lib/customisation.nix

```diff
@@ -106,9 +106,7 @@ rec {
       auto = builtins.intersectAttrs (builtins.functionArgs f) autoArgs;
       finalArgs = auto // args;
       pkgs = f finalArgs;
-      mkAttrOverridable = name: pkg: pkg // {
-        override = newArgs: mkAttrOverridable name (f (finalArgs // newArgs)).${name};
-      };
+      mkAttrOverridable = name: pkg: makeOverridable (newArgs: (f newArgs).${name}) finalArgs;
```

Comment:
No problem. There are `args`, `auto`, `autoArgs`, `finalArgs` and `newArgs` in one function! So in any case it is better to follow binding declarations than relying on mnemonics.